gammapy/maps/tests/test_wcs.py

```diff
@@ -296,3 +296,36 @@ def test_get_axis_index_by_name():
     assert geom.get_axis_index_by_name("Energy") == 0
     with pytest.raises(ValueError):
         geom.get_axis_index_by_name("time")
+
+test_axis1 = [MapAxis(nodes=(1,2,3,4), unit='TeV', node_type='center')]
```

Comment:
Also here, I would suggest to try and reduce the lines of test code if possible.
We already have many 1000s too much and it's hard to maintain.

Concretely, I think we shouldn't even try to exhaustively test astropy.wcs.WCS properties, but just rely that that compare works, and just have 1 or 2 explicit cases where it's the same and we get True and one where it's off by 1e-3 and False.
And the whole `@pytest.mark.parametrize` can be removed and the test cases in the test function, if there's only one line for each the test parametrisation doesn't by you anything, no?

Matter of taste - feel free to finish up as you like an merge.